src/session/client-certificate.c

```diff
@@ -127,6 +132,97 @@ unsigned long long get_proc_pid_start_time (int dirfd)
   return start_time;
 }
 
+/* Fallback for get_ws_proc_fd() on older kernels which don't support enough pidfd API */
+static int
+get_ws_proc_fd_pid_time (void)
```

Comment:
Reminder: don't forget to also make this take `int sockfd` if you take that suggestion.

Reply:
Done.